examples/cndpfwd/parse-args.c

```diff
@@ -76,6 +76,15 @@ process_callback(jcfg_info_t *j __cne_unused, void *_obj, void *arg, int idx)
                 if (f->xdp_uds == NULL)
                     CNE_ERR_RET("UDS handshake failed\n");
             }
+        } else if (!strncmp(obj.opt->name, FIB_RULES_TAG, nlen)) {
+            if (obj.opt->val.type == ARRAY_OPT_TYPE) {
+                f->fib_rules = malloc(obj.opt->val.array_sz * sizeof(char *));
+                uint16_t i   = 0;
```

Comment:
Move this line to after line 80 and remove the = 0; as it is not required. The forloop will set the value to zero.

Remove the {} from around the for loop as they are not required here and match the style.